src/components/LegoGallery/index.tsx

```diff
@@ -0,0 +1,157 @@
+    text: 'A standard interface for digital assets that can be either fungible or non-fungible tokens.',
+  },
+  {
+    title: 'LSP8 - Identifiable Digital Asset',
+    link: './tokens/LSP8-Identifiable-Digital-Asset',
+    lightImage: '/img/standards/legos/LSP8-lego.png',
+    darkImage: '/img/standards/legos/LSP8-lego.png',
+    text: 'A standard interface to represent identifiable digital assets, allowing tokens and NFTs to be uniquely traded and given specific metadata.',
+  },
+  {
+    title: 'LSP9 - Vault',
+    link: './universal-profile/lsp9-vault',
+    lightImage: '/img/standards/legos/LSP9-lego.png',
+    darkImage: '/img/standards/legos/LSP9-lego.png',
+    text: 'A version of an ERC725 smart contract that represents a blockchain vault.',
+  },
+  {
+    title: 'LSP10 - Received Vaults',
+    link: './universal-profile/lsp10-received-vaults',
+    lightImage: '/img/standards/legos/LSP10-lego.png',
+    darkImage: '/img/standards/legos/LSP10-lego.png',
+    text: 'A set of ERC725Y data keys to store the list of owned LSP9 Vaults.',
+  },
+  {
+    title: 'LSP12 - Issued Assets',
+    link: './universal-profile/lsp12-issued-assets',
+    lightImage: '/img/standards/legos/LSP12-lego.png',
+    darkImage: '/img/standards/legos/LSP12-lego.png',
+    text: 'A set of ERC725Y data keys to store the list of assets created or issued by a person, entity, organization, etc...',
+  },
+  {
+    title: 'LSP14 - Ownable 2 Steps',
+    link: './generic-standards/lsp14-ownable-2-step',
+    lightImage: '/img/standards/legos/LSP14-lego.png',
+    darkImage: '/img/standards/legos/LSP14-lego.png',
+    text: 'A module to allow a safer and more secure ownership management of a smart contract.',
+  },
+  {
+    title: 'LSP17 - Contract Extension',
+    link: './generic-standards/lsp17-contract-extension',
+    lightImage: '/img/standards/legos/LSP17-lego.png',
+    darkImage: '/img/standards/legos/LSP17-lego.png',
+    text: 'A module to extend functionalities of a smart contract by using other smart contract as "extension plugins", that can be added or removed overtime.',
+  },
+  {
+    title: 'LSP20 - Call Verification',
+    link: './generic-standards/lsp20-call-verification',
+    lightImage: '/img/standards/legos/LSP20-lego.png',
+    darkImage: '/img/standards/legos/LSP20-lego.png',
+    text: 'A module to make interactions with smart contracts easier, without the need to resolve and go through their `owner()` first.',
+  },
+  {
+    title: 'LSP23 - Linked Contracts Factory',
+    link: './generic-standards/lsp23-linked-contracts-factory',
+    lightImage: '/img/standards/legos/LSP23-lego.png',
+    darkImage: '/img/standards/legos/LSP23-lego.png',
+    text: 'Standard that define a factory pattern to deploy 2 contracts linked together at the pre-determined addresses, so that they can be redeployed across multiple chains.',
+  },
+];
+
+function LegoCard({ link, title, text, lightImage, darkImage }) {
```

Comment:
In my opinion, the boxes do not integrate well into the modern style of the docs. I would suggest adding something like these styles.

```css
    border-radius: 0.5rem;
    margin-left: 1rem;
    /* box-sizing: border-box; */
```
<img width="901" alt="pic-button" src="https://github.com/lukso-network/docs/assets/61689369/9cb52269-ba59-48d9-8a11-29f31fe95dc8">

Due to the `margin` you might have to set their `max-width`, so it still shows 4 per row. 

I would also remove the underline from the LSP descriptions, as the button is highlighted anyway.

Reply:
Ok good point here. Will look into this